labgraph/examples/rate.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserved.
+
+# Inspired by ROS Rate feature and this is a modified version of script below.
+# https://github.com/ros/ros_comm/blob/noetic-devel/clients/rospy/src/rospy/timer.py
+
+
+# Built-in imports
+import time
+import asyncio
+
+
+class Rate(object):
+    """
+    Convenience class for sleeping in a loop at a specified rate
+    """
+    
+    def __init__(self, hz):
```

Comment:
We shall using type hints in LabGraph codes.
Example:
def __init__(self, hz: float):